src/test/scala/com/cognite/sdk/scala/v1/SequencesTest.scala

```diff
@@ -264,4 +283,30 @@ class SequencesTest extends SdkTestSpec with ReadBehaviours with WritableBehavio
     )
     assert(limitDescriptionSearchResults.length == 1)
   }
+
+  it should "support search with dataSetIds" in {
+    val created = client.sequences.createFromRead(sequencesToCreate)
+    try {
+      val fromTime = created.map(_.createdTime).min
```

Comment:
See comments above. I think this should just be made part of the previous test, which tests search using several different parameters.

Reply:
> I think this should just be made part of the previous test, which tests search using several different parameters.

It is however searching some existing data, which does not work for my use case (there do not seem to be any data with `dataSetId`s). I can still merge it, but the test is quite different, IMHO.